### ISSUE TITLE
[Bug]: Package Discover: Argument #1 ($directory) must be of type string, null given

### ISSUE THREAD

**Issue by werksarztzentrum** [Laravel-Lang/common#193]
### Lang Package Name

laravel-lang/config

### Lang Package Version

1.4.0

### Laravel Framework Version

11.10

### PHP Version

8.2

### Dependencies

  "require-dev": {
        "laravel-lang/attributes": "^2.10.7",
        "laravel-lang/lang": "^15.5.1",
        "laravel-lang/publisher": "^16.4.0",
        ...
}

### Issue description

After upgrading to laravel-lang/config 1.4.0 we will get following error in all projects:

```
In ModelsData.php line 9:
                                                                                                                                                                                                                                         
  LaravelLang\Config\Data\Hidden\ModelsData::__construct(): Argument #1 ($directory) must be of type string, null given, called in /home/just/projects/dev-server/manager/vendor/laravel-lang/config/src/Services/Config.php on line 51  
                                                                                                                                                                                                                                         

Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1
```

### Steps to reproduce

Upgrading laravel-lang/config (1.2.0 => 1.4.0) with composer update

**Comment by werksarztzentrum** [comment]
Oh there are already opened issues, i will close it then.

**Comment by andrey-helldar** [comment]
The bug has been fixed in version [1.4.1](https://github.com/Laravel-Lang/config/releases/tag/1.4.1) of `laravel-lang/config` package.

Once again I apologize for the inconvenience. This was a bug that could not be detected during preliminary testing.

Thank you for your report!
